src/main/kotlin/com/cognifide/gradle/aem/common/instance/LocalInstanceManager.kt

```diff
@@ -497,9 +498,29 @@ class LocalInstanceManager(internal val aem: AemExtension) : Serializable {
         return killedInstances
     }
 
-    fun examine(instances: Collection<LocalInstance> = aem.localInstances) = base.examine(instances)
+    val examineEnabled = aem.obj.boolean {
+        convention(base.examineEnabled)
+        aem.prop.boolean("localInstance.examination.enabled")?.let { set(it) }
+    }
+
+    fun examine(instances: Collection<LocalInstance> = aem.localInstances) {
+        if (!examineEnabled.get()) {
+            return
+        }
+
+        base.examine(instances)
```

Comment:
Wouldn't be better to not have this return in the middle?
```
        if (examineEnabled.get()) {
            base.examine(instances)
        }
```

Reply:
This is about code style. My style prefers short returns instead of longly wrapped code blocks